grow_cluster_test.py

```diff
@@ -81,11 +81,8 @@ def add_nodes(self, add_node_cnt):
     def grow_cluster(self, cluster_target_size, stress_cmd):
         self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(),
                                     tester_obj=self)
-        # 60 minutes should be long enough for adding each node
-        nodes_to_add = cluster_target_size - self._cluster_starting_size
-
-        # 240 min - run nemesis with load in parallel after the cluster was expanded
-        duration = (60 * nodes_to_add) + 240
+        # default=1440 min (one day) if test_duration is not defined
+        duration = self.params.get('test_duration', default=1440)
```

Comment:
why do you set default that is different from the yaml?

Reply:
And why 1440?

Reply:
> why do you set default that is different from the yaml?

Because we have another test scale-450-500

Reply:
> And why 1440?

I don't know how long time is needed for 450-500 test. I just put longer time

Reply:
but isn't it using the same yaml? so it will get the same param from the yaml, not the default, isn't it?

Reply:
> but isn't it using the same yaml? so it will get the same param from the yaml, not the default, isn't it?

no, it's not the same yaml

Reply:
so change this default and set for the 500 nodes a different value.

Reply:
As I wrote above, I don't know how long time will take the 450-500 test. We didn't run it even once